JWPlayer-SDK-iOS-Demo/HelperExtensions.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  HelperExtensions.swift
+//  JWPlayer-SDK-iOS-Demo
+//
+//  Created by Amitai Blickstein on 3/26/19.
+//  Copyright © 2019 JWPlayer. All rights reserved.
+//
+
+import Foundation
+
+extension JWPlayerController {
```

Comment:
@AmitaiB do not extend the player controller. This logic should not be on the player controller, and it would be confusing to developers since at a first glance it appears that this property is exposed by the player controller.

Reply:
I'll just build it as if Autolayout works, and it'll be buggy until we release 3.3.0.